components/Carousel/Carousel.js

```diff
@@ -1,3 +1,39 @@
+var carouselDiv = document.createElement("div");
+let carousel = document.querySelector(".carousel-container");
+carouselDiv.classList.add("carousel");
+carousel.appendChild(carouselDiv);
+var left = document.createElement("div");
+left.classList.add("left-button");
+carouselDiv.appendChild(left);
+left.textContent="<";
+
+var img1 = document.createElement("img");
+img1.classList.add("show");
+img1.dataset["order"] = 1;
+img1.src = "/assets/carousel/mountains.jpeg";
+carouselDiv.appendChild(img1);
+
+var img2 = document.createElement("img");
+img2.dataset["order"] = 2;
+img2.src = "/assets/carousel/computer.jpeg";
+carouselDiv.appendChild(img2);
+
+var img3 = document.createElement("img");
+img3.dataset["order"] = 3;
+img3.src = "/assets/carousel/trees.jpeg";
+carouselDiv.appendChild(img3);
+
+var img4 = document.createElement("img");
+img4.dataset["order"] = 4;
+img4.src = "/assets/carousel/turntable.jpeg";
+carouselDiv.appendChild(img4);
+
+var right = document.createElement("div");
+right.classList.add("right-button");
+carouselDiv.appendChild(right);
+right.textContent=">";
+
+
```

Comment:
https://github.com/calebredd/Sprint-Challenge-Applied-Javascript/blob/6acc32ce28c2508d54f4a36db006d4f6e7c52a7d/components/Carousel/Carousel.js#L1-L31

Going forward, ECMA standards recommend not using var in future codebases, and recommend using const and let.  Generally, I start off with const and then if I notice something is going to be reassigned I switch it to let, if I don't catch it first.  This was a hard transition for me, so I understand, but something that is recommended due to the interesting issues that could arise because of var.